R/model-module-helpers.R

```diff
@@ -178,23 +178,17 @@ add_pmfs <- function(pmfs) {
   if (!is.list(pmfs)) {
     return(pmfs)
   }
-  lpmfs <- purrr::map_dbl(pmfs, length)
-  l <- sum(lpmfs)
-  conv <- rep(0, l)
-  conv[1:lpmfs[1]] <- pmfs[[1]]
-  for (s in 2:d) {
-    # P(Z = z) = sum_over_x(P(X = x) * P(Y = z - x)) # nolint
-    proc <- rep(0, l)
-    for (i in 1:l) {
-      for (j in 1:lpmfs[s]) {
-        if (i >= j) {
-          proc[i] <- proc[i] + pmfs[[s]][j] * conv[i - j + 1]
-        }
+  return(
```

Comment:
This is a stylistic choice and one I wouldn't usually take. Happy to leave as is after reflection.

Reply:
Would you prefer assigment to `conv` and then `return()`?

Reply:
Changed to explicit `return` but no assignment in https://github.com/epinowcast/epinowcast/pull/187/commits/bdf172ebeff5eb4b3882cdab6f0e4a3d8abe228f